### PR TITLE
Switch to the new gmock

### LINKED ISSUE #177: Switch to the latest GMock
Moved from GoogleCode to: https://github.com/google/googlemock

Which has now moved to part of GoogleTest:
https://github.com/google/googletest

### PR DESCRIPTION
**Pull request by nomis52**
Closes #177

### CODE REVIEW

File: install-gmock.sh [hunk 1]
```diff
@@ -1,8 +1,11 @@
...
+
+GTEST_DIRECTORY=gtest-read-only
+GTEST_REPO=https://github.com/google/googletest.git
+
+git clone $GTEST_REPO $GTEST_DIRECTORY
+cd $GTEST_DIRECTORY/googlemock
 autoreconf -fi
 ./configure
 make
-cd ..
```
Comment (peternewman) [inline install-gmock.sh]: I take it this is intentional?

**Comment by peternewman** [comment]
LGTM

File: install-gmock.sh (hunk 1 above)
Comment (nomis52) [inline install-gmock.sh]: Yup, this is run as a separate process, so the cwd doesn't affect the parent.

Comment (peternewman) [inline install-gmock.sh]: Okay

**Pull request merged**